grpc/src/compiler/cpp_generator.cc

```diff
@@ -201,15 +201,15 @@ void PrintHeaderClientMethodInterfaces(
         (*vars)["AsyncPrefix"] = async_prefix.prefix;
```

Comment:
This file is copied (and occasionally updated) from the GRPC project.. if it is not also fixed upstream, it will eventually be overwritten

Reply:
Opened an issue on gRPC for this: https://github.com/grpc/grpc/issues/19191

Reply:
Thanks!